### PR TITLE
Integrate dispatch frontend and API for end-to-end simulation visualization

### PR DESCRIPTION
**Pull request by gmarupilla**
Simulation dispatch working frontend

### CODE REVIEW

**Comment by gmarupilla** [comment]
@bilalshaikh42 I had to revert changes from #1229 because that PR only removed the prefix from main.ts but not from other places it was being used, all the dispatch apps and proxies were breaking due to that.

File: biosimulations/apps/dispatch-api/src/main.ts [hunk 1]
```diff
@@ -12,12 +12,14 @@ import { CustomOrigin } from '@nestjs/common/interfaces/external/cors-options.in
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  const globalPrefix = 'api';
```
Comment (bilalshaikh42) [inline biosimulations/apps/dispatch-api/src/main.ts]: I dont think we want to add this back, correct?

**Review by bilalshaikh42 — CHANGES_REQUESTED** [review]

**Comment by bilalshaikh42** [comment]
> @bilalshaikh42 I had to revert changes from #1229 because that PR only removed the prefix from main.ts but not from other places it was being used, all the dispatch apps and proxies were breaking due to that.

Let's remove the prefix. The only place that should have the url hardcoded is the URL library correct? 

The library can be updated as well.

**Comment by gmarupilla** [comment]
Removing 'api' as prefix was breaking dispatch frontend,Since the proxy expects 'api' and also all the urls in libs/urls. Ts still have 'api'.. If we are removing from one place, we need to remove it from all the places and fix proxy as well

File: biosimulations/apps/dispatch-api/src/main.ts (hunk 1 above)
Comment (gmarupilla) [reply biosimulations/apps/dispatch-api/src/main.ts]: removed

File: biosimulations/apps/dispatch-api/src/app/app.controller.ts [hunk 2]
```diff
@@ -105,7 +104,7 @@ export class AppController implements OnApplicationBootstrap {
     const simSpec: SimulationDispatchSpec = {
       authorEmail: bodyData.authorEmail,
       nameOfSimulation: bodyData.nameOfSimulation,
-      simulator: bodyData.simulator,
+      simulator: bodyData.simulator.toUpperCase(),
```
Comment (bilalshaikh42) [inline biosimulations/apps/dispatch-api/src/app/app.controller.ts]: The simulators at [biosimulators](http://biosimulators.org/simulators) will have all lowercase id's. We should be consistent between the two

**Review by bilalshaikh42 — CHANGES_REQUESTED** [review]

File: biosimulations/apps/dispatch-api/src/app/app.controller.ts (hunk 2 above)
Comment (bilalshaikh42) [reply biosimulations/apps/dispatch-api/src/app/app.controller.ts]: Just want to make sure this won't break anything on the backend. If not, I can make the change and then merge it in

**Comment by gmarupilla** [comment]
No, it’s not going to break anything at least for now. Eventually, I’ll plug-in simulator-api.

**Review by bilalshaikh42 — APPROVED** [review]

**Pull request merged**